### PR TITLE
Add note about relative links when discussing dir structure

### PR DESCRIPTION
**Pull request by m-cat**

### CODE REVIEW

File: source/includes/_uploading.md [hunk 1]
```diff
@@ -210,32 +210,42 @@ It is possible to upload a directory as a single piece of content. Doing this
 will allow you to address your content under one skylink, and access the files
 by their path. This is especially useful for webapps.
 
-For example, let's say you upload a directory with the following structure:
+For example, let's say you upload a Skynet app with the following simple structure:
```
Comment (kwypchlo) [inline source/includes/_uploading.md:213]:
```suggestion
For example, let's say you upload a web app with the following simple structure:
```

File: source/includes/_uploading.md [hunk 2]
```diff
@@ -210,32 +210,42 @@ It is possible to upload a directory as a single piece of content. Doing this
...
 
 <pre class="not_example">
-...portal.../...skylink...
-...portal.../...skylink.../dir2
+[portal url]/[skylink]/css
+[portal url]/[skylink]/js
 </pre>
 
-will download it as a .zip archive (by default) containing the directory contents.
+or `src` at `[portal url]/[skylink]`, will download it as a .zip archive (by
```
Comment (kwypchlo) [inline source/includes/_uploading.md:241]: `[portal url]/[skylink]` this should redirect to `[portal url]/[skylink]/` and load `index.html` file actually, right?

File: source/includes/_uploading.md [hunk 3]
```diff
@@ -210,32 +210,42 @@ It is possible to upload a directory as a single piece of content. Doing this
...
+default) containing the directory contents.
+
+<aside class="success">
+You can use relative links to refer to different files within a Skynet app. For
```
Comment (kwypchlo) [inline source/includes/_uploading.md:245]:
```suggestion
You can use relative links to refer to different files within a web app. For
```

**Review by kwypchlo — CHANGES_REQUESTED** [review]

File: source/includes/_uploading.md (hunk 2 above)
Comment (m-cat) [reply source/includes/_uploading.md:241]: Ah you're right, we don't download zip archives by default anymore.

**Pull request merged**